source/adapters/level_zero/command_buffer.cpp

```diff
@@ -217,30 +219,33 @@ ur_result_t calculateKernelWorkDimensions(
     break;
 
   default:
-    urPrint("urCommandBufferAppendKernelLaunchExp: unsupported work_dim\n");
+    logger::error("urCommandBufferAppendKernelLaunchExp: unsupported work_dim");
```

Comment:
here you removed the `\n`, above you left it as it was. My guess is you want to remove all these `\n` (I spotted at least several other).

Reply:
\n is added by logger so all \n are redundant